app/controllers/nyre/projects_controller.rb

```diff
@@ -37,19 +58,19 @@ def subsite_config
     # haaaaaaack to not reproduce templates
     def initialize(*args)
       super(*args)
-      self._prefixes << subsite_config['layout'] + '/projects' 
-      self._prefixes << subsite_config['layout']
+      self._prefixes << 'nyre/projects' 
+      self._prefixes << 'nyre'
       self._prefixes << '/catalog'
       self._prefixes.unshift "shared"
       self._prefixes.unshift ""
     end
 
     def set_view_path
       self.prepend_view_path('app/views/shared')
-      self.prepend_view_path('app/views/' + self.subsite_layout)
-      self.prepend_view_path(self.subsite_layout)
-      self.prepend_view_path('app/views/' + self.subsite_layout + '/projects')
-      self.prepend_view_path(self.subsite_layout + '/projects')
+      self.prepend_view_path('app/views/nyre')
+      self.prepend_view_path('nyre')
+      self.prepend_view_path('app/views/nyre/projects')
+      self.prepend_view_path('nyre/projects')
```

Comment:
Same note here about this being less dynamically generated than before, and instead relying on hard-coded "nyre".

Reply:
I'll add a commit for these.

Reply:
Added!

Reply:
Thanks!